src/lib/components/ProjectPage/ProjectPage.tsx

```diff
@@ -25,27 +26,30 @@ export const ProjectPage = ({ projectId, projectType }: IProjectPageProps) => {
     projectId,
     projectType,
   });
+
   return (
-    <PageLayout
-      title={title}
-      breadcrubms={[
-        APP_BREADCRUMB,
-        projectsTypeBreadCrumb,
-        {
-          text: isLoading ? '---' : projectBreadCrumbText,
-          id: ERoutesNames.Project,
-          navList: projectsNavList,
-        },
-      ]}
-    >
-      {isLoading ? (
-        <EmptyState
-          animation='pulse'
-          content={DISPLAY_TEXTS.he.tableStates[ETableStates.Loading]}
-        />
-      ) : (
-        <ProjectOverview />
-      )}
-    </PageLayout>
+    <SearchableContextProvider>
```

Comment:
i'm not sure we want to have this context so higher... it can cause a lot of unnecessary re-renders... why not wrap it around the table like you did with vendors...

Reply:
in case you save a block of code for later use , you need to add a comment that explain why the code is commented...
e.g. :
```
//TODO: decide later if we need those filters , (currently comment them out):
```

Reply:
you right.
if it wil be on the projetOverview it will be better? insted of wraping every table individualy...